src/MCPServer/lib/linkTaskManagerFiles.py

```diff
@@ -49,6 +49,9 @@ class linkTaskManagerFiles(LinkTaskManager):
     def __init__(self, jobChainLink, pk, unit):
         super(linkTaskManagerFiles, self).__init__(jobChainLink, pk, unit)
 
+        if jobChainLink.reloadFileList:
+            unit.reloadFileList()
```

Comment:
This used to happen in `createTasks`, just moving it here because it seems a better place.